### ISSUE TITLE
actionpack 8.0

### ISSUE THREAD

**Issue by quochien** [fringd/zipline#109]
Not sure if we can have update for Rails 8.0? 
Thanks in advance!

**Comment by SebastianPoell** [comment]
Yeah, currently, resolving the bundle for Rails 8 results in an error. There already is a PR #108, which could be used as a workaround. However, a new release would be awesome 👍

**Comment by pskl** [comment]
Currently blocked by this also!

**Comment by aleksejkuzmin** [comment]
would be nice to merge it, also faced with this issue

**Comment by AdnanTheExcellent** [comment]
Also faced with being unable to upgrade to rails 8.0

**Comment by egracens** [comment]
Same here.

**Comment by Spone** [comment]
This is fixed since #108 has been merged.
